components/infobox/commons/infobox_team.lua

```diff
@@ -187,13 +211,13 @@ function Team:_setLpdbData(args, links)
 			Links.makeFullLinksForTableItems(links or {}, 'team')
 		),
 	}
+	lpdbData.extradata = {}
+	for year, earningsOfYear in pairs(_earnings) do
+		lpdbData.extradata['earningsin' .. year] = earningsOfYear
+	end
```

Comment:
This would add `extradata.earningsintotal` as well if I'm reading the code correctly.

Reply:
yeah, thx